crates/test-programs/Cargo.toml

```diff
@@ -10,20 +10,24 @@ license = "Apache-2.0 WITH LLVM-exception"
 [build-dependencies]
 cfg-if = "1.0"
 cargo_metadata = "0.15.3"
+wit-component = "0.7.4"
+heck = "0.4.0"
 
 [dev-dependencies]
+anyhow = { workspace = true }
+tempfile = "3.1.0"
+test-log = { version = "0.2", default-features = false, features = ["trace"] }
```

Comment:
Oh? I've not seen this dependency before, what does it do?

Reply:
It is a wrapper over `#[test]` (or e.g. `#[tokio::test]` if you pass it in as an argument) which takes care of installing tracing-subscriber with an env filter before starting your test. It gets rid of a Once cell we used to do that manually.